### PR TITLE
Add section about active and inactive contributors

### PR DESCRIPTION
**Pull request by friedger**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -122,3 +122,18 @@ from the main (upstream) repository:
 [github]: https://github.com/gdg-x
 [pluspage]: https://google.com/+GDGXProject
 [js-style-guide]: http://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml
+
+## <a name="members"></a> GDG[x] contributors
+Everybody is invite to dive into the code, explore, learn, add issues and send pull requests.
```
Comment (Splaktar) [inline CONTRIBUTING.md]: Everybody is invited to dive into the code, explore, learn, open issues, and send pull requests.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -122,3 +122,18 @@ from the main (upstream) repository:
...
+
+### Teams
+Each project has a *team of contributors* that can merge pull requests. These team members should have a good understanding of the project to answer questions and comment on pull requests. These contributors are members of the corresponding github project team.
```
Comment (Splaktar) [inline CONTRIBUTING.md]: and review pull requests.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -122,3 +122,18 @@ from the main (upstream) repository:
...
+
+One or two members are the *maintainers* of the project and publish releases. Maintainers are responsible for the road map and the state of the project in general.
+
+Maintainers are listed by name/github handle in the readme of the project.
+
+### Inactive contributors
+Contributors can retire by [leaving the github project team](https://github.com/orgs/gdg-x/teams) and [join the alumni team](https://github.com/orgs/gdg-x/teams/alumni). Alternatively, they can [leave the organisation](https://github.com/settings/organizations).
```
Comment (Splaktar) [inline CONTRIBUTING.md]: [joining the alumni team]

Comment (Splaktar) [inline CONTRIBUTING.md]: [leave the organization]

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -122,3 +122,18 @@ from the main (upstream) repository:
...
+
+Contributors who were inactive for more than a year are contacted by the project maintainers and asked to retire. If the inactive contributor can't be reached the contributor is moved to the alumni team by one of the [GDG[x] admin organizers team](https://github.com/orgs/gdg-x/teams/og) members.
```
Comment (Splaktar) [inline CONTRIBUTING.md]: Contributors who are inactive for more than a year will be contacted by the project maintainers and asked if they can become active again or retire.

Comment (Splaktar) [inline CONTRIBUTING.md]: [GDG[x] admin organizers](https://github.com/orgs/gdg-x/teams/og).

Comment (Splaktar) [inline CONTRIBUTING.md]: We should probably add a note about Maintainers who are inactive. Should we make it 6 months for maintainers? Should we mention here what happens to a project when it no longer has a maintainer?

**Comment by joshuawoodward** [comment]
You guys are doing a great job!!

File: CONTRIBUTING.md (hunk 3 above)
Comment (friedger) [inline CONTRIBUTING.md]: not sure what is wrong with organisation (:-p). Should we add into code of conduct that the language is en-US?

**Comment by friedger** [comment]
Comments addressed! @Splaktar

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -122,3 +122,22 @@ from the main (upstream) repository:
...
+Each project has a *team of contributors* that can merge pull requests. These team members should have a good understanding of the project to answer questions and review pull requests. These contributors are members of the corresponding github project team.
+
+One or two members are the *maintainers* of the project and publish releases. Maintainers are responsible for the road map and the state of the project in general.
+
+Maintainers are listed by name/github handle in the readme of the project.
+
+### Inactive contributors
+Contributors can retire by [leaving the github project team](https://github.com/orgs/gdg-x/teams) and [joining the alumni team](https://github.com/orgs/gdg-x/teams/alumni). Alternatively, they can [leave the organization](https://github.com/settings/organizations).
+
+Contributors who are inactive for more than a year will be contacted by the project maintainers and asked if they can become active again or retire.
```
Comment (tasomaniac) [inline CONTRIBUTING.md:131]: I think we should contact them earlier. Like in 6 months. We can just send a template email. They can be moved to alumni team at the end of 1 year of inactivity.

Comment (friedger) [inline CONTRIBUTING.md:131]: I think you are too ambitious :-)

**Comment by tasomaniac** [comment]
What is holding us here? We should merge this.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -122,3 +122,22 @@ from the main (upstream) repository:
 [github]: https://github.com/gdg-x
 [pluspage]: https://google.com/+GDGXProject
 [js-style-guide]: http://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml
+
+## <a name="members"></a> GDG[x] contributors
+Everybody is invited to dive into the code, explore, learn, open issues, and send pull requests.
+
+### Teams
+Each project has a *team of contributors* that can merge pull requests. These team members should have a good understanding of the project to answer questions and review pull requests. These contributors are members of the corresponding github project team.
```
Comment (Splaktar) [inline CONTRIBUTING.md]: Contributors are members of the corresponding GitHub project team.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -122,3 +122,22 @@ from the main (upstream) repository:
...
+
+One or two members are the *maintainers* of the project and publish releases. Maintainers are responsible for the road map and the state of the project in general.
```
Comment (Splaktar) [inline CONTRIBUTING.md]: One or two members are the _maintainers_ of the project. Maintainers are responsible for the roadmap, milestones, and the state of the project in general.  They tag and publish releases.

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -122,3 +122,22 @@ from the main (upstream) repository:
...
+
+Maintainers are listed by name/github handle in the readme of the project.
```
Comment (Splaktar) [inline CONTRIBUTING.md]: Maintainers are listed and link by name/handle in the readme of each project.

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -122,3 +122,22 @@ from the main (upstream) repository:
...
+
+### Inactive contributors
+Contributors can retire by [leaving the github project team](https://github.com/orgs/gdg-x/teams) and [joining the alumni team](https://github.com/orgs/gdg-x/teams/alumni). Alternatively, they can [leave the organization](https://github.com/settings/organizations).
```
Comment (Splaktar) [inline CONTRIBUTING.md]: [leaving the GitHub project team]

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -122,3 +122,22 @@ from the main (upstream) repository:
...
+
+Contributors who are inactive for more than a year will be contacted by the project maintainers and asked if they can become active again or retire. 
+If the inactive contributor can't be reached the contributor is moved to the alumni team by one of the [GDG[x] admin organizers](https://github.com/orgs/gdg-x/teams/og) members.
```
Comment (Splaktar) [inline CONTRIBUTING.md]: Probably don't need to say `admin organizers members`. Probably `admin organizers` is enough.

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -122,3 +122,22 @@ from the main (upstream) repository:
...
+
+### Inactive maintainers
+Maintainers who are inactive for more than six months will be contacted by one of the GDG[x] admin organizers and ask whether they can become active again or retire (see inactive contributors). If they retire and no other maintainers can be found the project is deprecated.
```
Comment (Splaktar) [inline CONTRIBUTING.md]: `and asked` and `the project is marked [DEPRECATED].`

**Pull request merged**